x-pack/legacy/plugins/canvas/i18n/components.ts

```diff
@@ -926,7 +926,7 @@ export const ComponentStrings = {
   Toolbar: {
     getEditorButtonLabel: () =>
       i18n.translate('xpack.canvas.toolbar.editorButtonLabel', {
-        defaultMessage: 'Expression editor',
+        defaultMessage: 'Expression Editor',
```

Comment:
Might want to check with @ryankeairns about this change since I think there is some copy style we're trying to standardize around

Reply:
Thanks for letting me know! I've reverted this change

Reply:
Yep, we use sentence casing so `Expression editor` is correct. There are certain cases where we use title casing if it is a proper noun (product name), but that is the exception rather than the norm.

Reply:
Thanks! I've fixed the text, so that it's using sentence casing.